pyenv-win/libexec/pyenv-install.vbs

```diff
@@ -71,30 +71,44 @@ Sub download(params)
     DownloadFile params(LV_URL), params(IP_InstallFile)
 End Sub
 
-Function deepExtract(params)
+Function deepExtract(params, web)
     ' WScript.echo "kkotari: pyenv-install.vbs deepExtract..!"
-    Dim webCachePath
+    Dim cachePath
     Dim installPath
-    webCachePath = strDirCache &"\"& params(LV_Code) &"-webinstall"
+    cachePath = strDirCache &"\"& params(LV_Code)
+    If web Then
+        cachePath = cachePath &"-webinstall"
+    End If
     installPath = params(IP_InstallPath)
     deepExtract = -1
 
-    If Not objfs.FolderExists(webCachePath) Then
-        deepExtract = objws.Run(""""& params(IP_InstallFile) &""" /quiet /layout """& webCachePath &"""", 0, True)
+    If Not objfs.FolderExists(cachePath) Then
+        deepExtract = objws.Run(""""& params(IP_InstallFile) &""" /quiet /layout """& cachePath &"""", 0, True)
         If deepExtract Then
-            WScript.Echo ":: [Error] :: error using web installer."
+            WScript.Echo ":: [Error] :: error extracting the web portion from the installer."
             Exit Function
         End If
+        If Not web Then
+            deepExtract = objws.Run(""""& strDirWiX &"\dark.exe"" -x """& cachePath &""" """& params(IP_InstallFile) &"""", 0, True)
```

Comment:
`WiX/dark` does not extract all `msi` files.

The following command

```
bin\WiX\dark.exe -x output python-xxx.exe
```

does not extract `*_d.msi` for Python debug build and debug symbols.

while

```
python-xxx.exe /layout output
```

does.